src/content/docs/style-guide/processes-procedures/tech-writer-workflow.mdx

```diff
@@ -22,7 +22,7 @@ Once GitHub Desktop is set up, navigate to the [Docs Site repository](https://gi
 
 ## Run the site locally
 
-You will use the Terminal to build the site locally. This will allow you to preview changes before opening a Pull Request. While it's' **highly** reccomended to build the site locally, this is technically an optional step. before opening  While running, the site will reflect any local changes made. Node and Yarn are tools we use to build the site on your local machine.
+You will use the Terminal to build the site locally. This will allow you to preview changes before opening a Pull Request. While it's **highly** reccomended to build the site locally, this is technically an optional step. Once the local site is built, it will automatically reflect any local changes made. Node and Yarn are tools used to build the site on your local machine.
```

Comment:
Typo: "reccomended"

I'd rewrite this paragraph to remove the passive voice. And the "will"s :)